schema/deploy/types/budget_item_category.sql

```diff
@@ -0,0 +1,12 @@
+-- Deploy cif:types/budget_item_category to pg
+
+begin;
+
+create type cif.budget_item_category as enum (
+  'general',
+  'materials, supplies, equipment and other items'
```

Comment:
This should be a table, not an enum. An enum would not be editable by users, and we want them to be able to configure this list